products/compute/api.yaml

```diff
@@ -1329,43 +1548,238 @@ objects:
+            name: 'interval'
+            description: |
+              Time interval between ejection sweep analysis. This can result in both new
+              ejections as well as hosts being returned to service. Defaults to 10 seconds.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'seconds'
+                required: true
+                description: |
+                  Span of time at a resolution of a second. Must be from 0 to 315,576,000,000
+                  inclusive.
+              - !ruby/object:Api::Type::Integer
+                name: 'nanos'
+                description: |
+                  Span of time that's a fraction of a second at nanosecond resolution. Durations
+                  less than one second are represented with a 0 `seconds` field and a positive
+                  `nanos` field. Must be from 0 to 999,999,999 inclusive.
+          - !ruby/object:Api::Type::Integer
+            name: 'maxEjectionPercent'
+            default_value: 10
+            description: |
+              Maximum percentage of hosts in the load balancing pool for the backend service
+              that can be ejected. Defaults to 10%.
+          - !ruby/object:Api::Type::Integer
+            name: 'successRateMinimumHosts'
+            default_value: 5
+            description: |
+              The number of hosts in a cluster that must have enough request volume to detect
+              success rate outliers. If the number of hosts is less than this setting, outlier
+              detection via success rate statistics is not performed for any host in the
+              cluster. Defaults to 5.
+          - !ruby/object:Api::Type::Integer
+            name: 'successRateRequestVolume'
+            default_value: 100
+            description: |
+              The minimum number of total requests that must be collected in one interval (as
+              defined by the interval duration above) to include this host in success rate
+              based outlier detection. If the volume is lower than this setting, outlier
+              detection via success rate statistics is not performed for that host. Defaults
+              to 100.
+          - !ruby/object:Api::Type::Integer
+            name: 'successRateStdevFactor'
+            default_value: 1900
+            description: |
+              This factor is used to determine the ejection threshold for success rate outlier
+              ejection. The ejection threshold is the difference between the mean success
+              rate, and the product of this factor and the standard deviation of the mean
+              success rate: mean - (stdev * success_rate_stdev_factor). This factor is divided
+              by a thousand to get a double. That is, if the desired factor is 1.9, the
+              runtime value should be 1900. Defaults to 1900.
+      # 'port' is deprecated
+      - !ruby/object:Api::Type::String
+        name: 'portName'
```

Comment:
We can't set the scheme to `EXTERNAL` thought, right? So this can likely be omitted?

Reply:
Right. [These docs](https://cloud.google.com/load-balancing/docs/choosing-load-balancer) suggest that `EXTERNAL` regional load balancing systems are possible, but the API currently rejects requests to create a `RegionBackendService` with `loadBalancingScheme` set to `EXTERNAL` with a message ~= "not yet supported".